### ISSUE TITLE
add info about bucket encryption

### ISSUE THREAD

**Issue by kdaily** [Sage-Bionetworks/synapseDocs#398]
We can use default AES256 (master AWS S3 keys) encryption with Synapse external storage locations with no other modifications.

**Comment by vortexing** [comment]
Is there more to this?  OR this is to come?  I just checked out the docs for custom S3 storage and it looks great so far, but we will always have encrypted buckets so making it clear that that won't matter (which is what I think this is saying) or if it does matter, how to deal with it would be great.  I'll be picking this back up this summer.  Will report back.

**Comment by kdaily** [comment]
@vortexing, nope, nothing more except to explicitly mention it works. If you set up a bucket using our instructions, but also turn on the default encryption (master AWS S3 keys) it will work out of the box. **It will NOT work with KMS (Key Management Service) encryption.**

**Comment by kdaily** [comment]
@vortexing we are considering turning it on by default in Synapse storage:

https://sagebionetworks.jira.com/browse/PLFM-4955
